src/NodeManipulator/ClassConstManipulator.php

```diff
@@ -25,8 +26,8 @@ public function __construct(
 
     public function hasClassConstFetch(ClassConst $classConst, ClassReflection $classReflection): bool
     {
-        $class = $this->betterNodeFinder->findParentType($classConst, Class_::class);
-        if (! $class instanceof Class_) {
+        $class = $this->betterNodeFinder->findParentByTypes($classConst, [Class_::class, Enum_::class]);
+        if ($class === null) {
             return false;
         }
```

Comment:
this check can just be removed, the `ClassReflection->getAncestors()` already check current class and parent class.

```suggestion
```